src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_java/java_adexchangebuyer.v1.4.json.baseline

```diff
@@ -1650,6 +1652,7 @@ import java.security.GeneralSecurityException;
 public class AdExchangeBuyerExample {
   public void run() throws IOException, GeneralSecurityException {
     // TODO: Assign values to the desired fields of `requestBody`.
+    // Only the assigned fields will be changed.
```

Comment:
I was more worried about HTTP `patch` being matched to a non-`patch` library method, where the comment might be puzzling.

OK.